docs/customising/make_a_new_theme.md

```diff
@@ -191,24 +203,19 @@ put the URL of your theme in your `THEME_URLS` config setting for your
 production server, just like you have done for this development one).
 
 There are different ways to commit your changes, but here's one method:
+<div class="attention-box helpful-hint">
+  Note that <em>inside</em> your Vagrant virtual machine, you won't be able to
+  access GitHub with your user settings, and so on. So you will need to do this
+  from "outside" the VM. This can trip you up because the command shell looks
+  very similar when you are inside a <code>vagrant&nbsp;ssh</code> session,
+  and when you are not.
+</div>
```

Comment:
My bad, it's because Jekyll ignores markup inside an HTML block. IIRC there is a workaround but the 100% works every time method is to write out the code the old fashioned way.

> On 1 Apr 2016, at 14:55, Gareth Rees <notifications@github.com> wrote:
> 
> In docs/customising/make_a_new_theme.md:
> 
> > @@ -191,24 +203,19 @@ put the URL of your theme in your `THEME_URLS` config setting for your
> >  production server, just like you have done for this development one).
> >  
> >  There are different ways to commit your changes, but here's one method:
> > +<div class="attention-box helpful-hint">
> > +  Note that <em>inside</em> your Vagrant virtual machine, you won't be able to
> > +  access GitHub with your user settings, and so on. So you will need to do this
> > +  from "outside" the VM. This can trip you up because the command shell looks
> > +  very similar when you are inside a <code>vagrant&nbsp;ssh</code> session,
> > +  and when you are not.
> > +</div>
> This isn't rendering for me either:
> 
> 
> 
> The code looks correct, so might be an issue with my jekyll or something.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>